libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -838,6 +856,45 @@ void _IotMqtt_DecrementConnectionReferences( _mqttConnection_t * pMqttConnection
 
 /*-----------------------------------------------------------*/
 
+static uint32_t getTimeMs( void )
+{
+    uint32_t timeMs;
+
+    return ( uint32_t ) IotClock_GetTimeMs();
+}
+
+/*-----------------------------------------------------------*/
+
+static int32_t transportSend( NetworkContext_t networkContext,
```

Comment:
Can the NetworkContext_t be containing the network interface? With that we can avoid the look up for the network interface